appengine-java8/bigtable/README.md

```diff
@@ -29,7 +29,7 @@ Then you need to [Create a Cloud Bigtable Instance](https://cloud.google.com/big
 
 ### Deploy to App Engine Standard for Java 8
 
-    mvn -Dbigtable.projectID=PROJECTID -Dbigtable.instanceID=INSTANCEID appengine:deploy
+    mvn -Dbigtable.projectID=PROJECTID -Dbigtable.instanceID=INSTANCEID package appengine:deploy 
```

Comment:
Why doesn't this require the -Dapp.deploy.projectId=<your-project-id>
 

 ??

Reply:
It is set using the bigtable project Id because why duplicate.